tooling/lsp/src/visibility.rs

```diff
@@ -0,0 +1,13 @@
+use noirc_frontend::{ast::ItemVisibility, hir::def_map::ModuleId};
+
+pub(super) fn is_visible(
+    visibility: ItemVisibility,
+    current_module: ModuleId,
+    target_module: ModuleId,
+) -> bool {
+    match visibility {
+        ItemVisibility::Public => true,
+        ItemVisibility::Private => false,
```

Comment:
This isn't always the case as you can import a private item from a submodule of where it's defined.

Looks like we'd have to make larger changes in order to suggest private items for auto-import so we can just create an issue for this.

Reply:
Makes sense. Captured in #5879